Mail/Template.php

```diff
@@ -205,10 +212,18 @@ public function loadDefault($templateId)
     public function getProcessedTemplate(array $variables = [])
     {
         $processor = $this->getTemplateFilter()
-            ->setUseSessionInUrl(false)
             ->setPlainTemplateMode($this->isPlain())
             ->setIsChildTemplate($this->isChildTemplate())
             ->setTemplateProcessor([$this, 'getTemplateContent']);
+        /**
+         * Use session in url
```

Comment:
Replace "Use session in url" to "After Magento v2.3.5 session_id is removed from URLs"